modules/op-monitoring-pipeline-run-status-using-pipelines-as-code.adoc

```diff
@@ -0,0 +1,76 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: REFERENCE
+[id="monitoring-pipeline-run-status-using-pipelines-as-code_{context}"]
+= Monitoring pipeline run status using {pac} 
+
+[role="_abstract"]
+Depending on the context and supported tools, you can monitor the status of a pipeline run in different ways.
+
+[discrete]
+.Status on GitHub Apps
+When a pipeline run finishes, the status is added in the *Check* tabs with limited information on how long each task of your pipeline took, and the output of the `tkn pipelinerun describe` command.
+
+[discrete]
+.Log error snippet
+When {pac} detects an error in one of the task of a pipeline, a small snippet consisting of the last 3 lines in the task breakdown of the first failed task is displayed.
+
+[NOTE]
+====
+{pac} avoids leaking secrets by looking into the pipeline run and replacing secret values with hidden characters. However, {pac} cannot hide secrets coming from workspaces and envFrom source.
+====
+
+[discrete]
+.Annotations for log error snippets
```

Comment:
This label is not appearing properly in the preview. The heading appears as part of the important admonition and does not look correct.

Reply:
Comment addressed!